packages/luna/src/locales/ar-eg.json

```diff
@@ -447,6 +447,7 @@
   "settings.notifications.lesson-date.description": "سيتم اشعارك بميعاد الدرس قبله بيوم و قبله بساعه",
   "settings.notifications.messages.title": "إشعارات الرسائل",
   "settings.notifications.messages.description": "سوف يتم اشعارك بجميع الرسائل الواردة",
+  "settings.notifications.comming-soon": "سوف تتوفر هذه الخاصية قريبا",
```

Comment:
We can play with words to make it more appeal to the end user by re-phrase it to "سوف يكون بإمكانك تخصص إشعاراتك بمجرد تفعيل هذه الخاصية قريبا"

Let me know what you think? 


---- 

I think the text i introduced is a little bit long so it is better to use it as a *description* instead of a *title* the new title can be "قريبا 🎉".